src/actions/index.js

```diff
@@ -0,0 +1,15 @@
+import data from '../data/events.json';
+
+export function initializeEvents() {
+  return {
+    type: 'INITIALIZE_EVENT_DATA',
+    data,
```

Comment:
@GameKyuubi @bill-feng 
Should we change `data` to `events`? It'll be more clearer?

Reply:
I personally think the name of the object we pass to the reducer should match the name of the field in the state, but it depends on what you guys think.